src/network/intercept.ts

```diff
@@ -97,17 +99,19 @@ export async function isBotMsg(
             if (isMsgAndHasText || isNotMsg) {
               didEmit = await emitMessageToBot(
                 msg,
-                botInTribe.dataValues,
+                botInTribe.dataValues, // HELP dataValues?
```

Comment:
So I think `dataValues` is type `any` since its up to the botowner for what they want passed there

Reply:
Really `any`? Does `{ [k: string]: any }` fit as well?

Reply:
I think that should work aswell

Reply:
how is `botInTribe.dataValues` passed as a bot to `emitMessageToBot`
is ChatBot.dataValues a ChatBot?

Reply:
Yup `dataValues` is a `{ [k: string]: any }`, its the actual values in the Model returns from Sequelize. ChatBot.dataValues is a ChatBot

Reply:
does it also have the same values?
if so, why is botInTribe.dataValues passed and not botInTribe?

Reply:
botInTribe is a sequelize model so it also has other stuff on it, like `update` method etc.